modules/terraform/init_test.go

```diff
@@ -27,7 +27,7 @@ func TestInitBackendConfig(t *testing.T) {
 	options := &Options{
 		TerraformDir: testFolder,
 		BackendConfig: map[string]interface{}{
-			"path": remoteStateFile,
+			remoteStateFile: nil,
```

Comment:
I think this still needs to be what it was since this is updating the `local` backend's `path` key. Can you actually create a new test that is doing something similar, but with the backend config stored in a file?